app/controllers/jobs_controller.rb

```diff
@@ -39,6 +39,20 @@ def run
     end
   end
 
+  def retry_queued
+    @jobs = Delayed::Job.where('attempts > 1')
```

Comment:
So you might want to use an `scope.update_all run_at: Time.now, attempts: 0` to do the update in SQL in the DB.